### PR TITLE
FIX-BESTPRACTICES: asr-oms-monitoring

### PR DESCRIPTION
**Pull request by prem-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: demos/asr-oms-monitoring/azuredeploy.parameters.us.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
         },
         "location": {
             "value": "usgovvirginia"
+        },
+        "psModulesUri": {
+            "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```
Comment (bmoore-msft) [inline demos/asr-oms-monitoring/azuredeploy.parameters.us.json:18]: if this is required, then set this as the defaultValue in the template and remove this from here

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: demos/asr-oms-monitoring/azuredeploy.parameters.us.json (hunk 1 above)
Comment (prem-spektra) [reply demos/asr-oms-monitoring/azuredeploy.parameters.us.json]: After making the required modifications, the Best Practices violation tag has been added with an  error log: Found hardcoded reference to core.windows.net Line: 46, Column: 60

Comment (badri-spektra) [reply demos/asr-oms-monitoring/azuredeploy.parameters.us.json]: If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?

Comment (bmoore-msft) [reply demos/asr-oms-monitoring/azuredeploy.parameters.us.json]: it's a bug in the TTK

**Pull request merged**